### ISSUE TITLE
SBERT model does not use true doys, but days since first observation offset by doy of first image.

### ISSUE THREAD

**Issue by Florian-Katerndahl** [Florian-Katerndahl/sits-dl#2]
https://github.com/ChrSchiller/forest_decline/blob/ec1a49b31d206818207183ffa4aa148ffef962fe/08_sits_bert_prediction_on_force_tiles.py#L189-L192

**Comment by Florian-Katerndahl** [comment]
Now the model throws an error. Great

```
Traceback (most recent call last):
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/bin/inference", line 6, in <module>
    sys.exit(main())
  File "/home/eouser/git-repos/sits-dl/apps/inference.py", line 267, in main
    output_torch[r_start:r_end, c_start:c_end] = inference_model.predict(
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/lib/python3.9/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/eouser/git-repos/sits-dl/sits_dl/sbert.py", line 304, in predict
    self.forward(
  File "/home/eouser/git-repos/sits-dl/sits_dl/sbert.py", line 271, in forward
    x = self.sbert(x, doy, mask)
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/eouser/git-repos/sits-dl/sits_dl/sbert.py", line 251, in forward
    x = self.embedding(input_sequence=x, doy_sequence=doy)
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/eouser/git-repos/sits-dl/sits_dl/sbert.py", line 82, in forward
    x[i, :, self.embed_size :] = self.position(
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/eouser/.cache/pypoetry/virtualenvs/sits-dl-JSq1D_Dn-py3.9/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/eouser/git-repos/sits-dl/sits_dl/sbert.py", line 40, in forward
    return self.pe[doy, :]
IndexError: index 2074 is out of bounds for dimension 0 with size 1826
```
2074 is a calculated Doy according to my interpretation of Chris' code.

**Comment by Florian-Katerndahl** [comment]
Should be fixed by #[d24245f](https://github.com/Florian-Katerndahl/sits-dl/commit/d24245f7476e21fa9760a4b8eae0145ba187edfe)
